### PR TITLE
Initializing new portal application

### PR DESCRIPTION
**Pull request by jourdiw**
## Issue

https://gravitee.atlassian.net/browse/APIM-4245

## Description

Initialize the bare bones for the new Dev Portal

Next PRs:
- add Storybook
- add schematics
- enable Chromatic?
- add new Portal to current Portal docker
- add instructions for adding a language to readme
- add Husky

## Additional context

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-zbnvzrtsvh.chromatic.com)

### CODE REVIEW

**Comment by jourdiw** [comment]
I will squash all the commits once it is ✅

File: gravitee-apim-portal-webui-next/angular.json [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        "sourceLocale": "en-US"
+      },
+      "root": "",
+      "sourceRoot": "src",
+      "prefix": "app",
+      "architect": {
+        "build": {
+          "builder": "@angular-devkit/build-angular:application",
+          "options": {
+            "outputPath": "../gravitee-apim-portal-webui/dist/next",
```
Comment (jourdiw) [inline gravitee-apim-portal-webui-next/angular.json:23]: @phiz71 Does this match what you have for the Docker branch?

File: gravitee-apim-portal-webui-next/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# GraviteeApimPortalWebuiV2
```
Comment (jansobczynski) [inline gravitee-apim-portal-webui-next/README.md]: WebuiNext

File: gravitee-apim-portal-webui-next/README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+Run `ng generate component component-name` to generate a new component. You can also use `ng generate directive|pipe|service|class|guard|interface|enum|module`.
+
+## Build
+
+Run `ng build` to build the project. The build artifacts will be stored in the `dist/` directory.
+
+## Running unit tests
+
+Run `ng test` to execute the unit tests via [Karma](https://karma-runner.github.io).
```
Comment (jansobczynski) [inline gravitee-apim-portal-webui-next/README.md]: AFAIK we use Jest

File: gravitee-apim-portal-webui-next/angular.json [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+          "defaultConfiguration": "development"
+        },
+        "extract-i18n": {
+          "builder": "@angular-devkit/build-angular:extract-i18n",
+          "options": {
+            "buildTarget": "gravitee-apim-portal-webui-next:build"
+          }
+        },
+        "test": {
+          "builder": "@angular-devkit/build-angular:karma",
```
Comment (jansobczynski) [inline gravitee-apim-portal-webui-next/angular.json]: Is this correct since we are not using karma?

**Review by jansobczynski — APPROVED** [review]

**Review by phiz71 — APPROVED** [review]

**Pull request merged**